SPECS/mariner-rpm-macros/macros.check

```diff
@@ -1,4 +1,3 @@
-# RPM macros for overriding the default behavior of %check to be nonfatal.
```

Comment:
This needs to wait, we need to make this change inline with the tools/pipelines. (Tools still expect this legacy behaviour and will scan for the `"====== CHECK DONE %{name}. EXIT STATUS $?"` I think).

Reply:
ack, i removed this commit from this pr and i'll open a separate PR later for the check-related changes